src/actions/ranking-actions.js

```diff
@@ -0,0 +1,290 @@
+    showSuccessMessage,
+    authErrorHandler
+} from 'openstack-uicore-foundation/lib/methods';
+
+//Rating Types
+
+export const RECEIVE_RATING_TYPE = 'RECEIVE_RATING_TYPE';
+export const RESET_RATING_TYPE_FORM = 'RESET_RATING_TYPE_FORM';
+export const UPDATE_RATING_TYPE = 'UPDATE_RATING_TYPE';
+export const RATING_TYPE_UPDATED = 'RATING_TYPE_UPDATED';
+export const RATING_TYPE_ADDED = 'RATING_TYPE_ADDED';
+export const RATING_TYPE_DELETED = 'RATING_TYPE_DELETED';
+export const RATING_TYPE_SCORE_TYPE_REMOVED = 'RATING_TYPE_SCORE_TYPE_REMOVED';
+export const RATING_TYPE_SCORE_TYPE_ADDED = 'RATING_TYPE_SCORE_TYPE_ADDED';
+export const RATING_TYPE_SCORE_TYPE_ORDER_UPDATED = 'RATING_TYPE_SCORE_TYPE_ORDER_UPDATED';
+
+export const getRatingType = (ratingTypeId) => (dispatch, getState) => {
+
+    const {loggedUserState, currentSummitState, currentSelectionPlanState} = getState();
+    const {accessToken} = loggedUserState;
+    const {currentSummit} = currentSummitState;
+    const currentSelectionPlan = currentSelectionPlanState.entity;
+
+    dispatch(startLoading());
+
+    const params = {
+        access_token: accessToken,
+        order: '+order',
+        expand: 'score_types,selection_plan'
+    };
+
+    return getRequest(
+        null,
+        createAction(RECEIVE_RATING_TYPE),
+        `${window.API_BASE_URL}/api/v1/summits/${currentSummit.id}/selection-plans/${currentSelectionPlan.id}/track-chair-rating-types/${ratingTypeId}`,
+        authErrorHandler
+    )(params)(dispatch).then(() => {
+            dispatch(stopLoading());
+        }
+    );
+};
+
+export const resetRatingTypeForm = () => (dispatch, getState) => {
+    dispatch(createAction(RESET_RATING_TYPE_FORM)({}));
+};
+
+export const saveRatingType = (entity) => (dispatch, getState) => {
+
+    const {loggedUserState, currentSummitState, currentSelectionPlanState} = getState();
+    const {accessToken} = loggedUserState;
+    const {currentSummit} = currentSummitState;
+    const currentSelectionPlan = currentSelectionPlanState.entity;
+
+    dispatch(startLoading());
+
+    const normalizedEntity = normalizeEntity(entity);
+
+    if (entity.id) {
+        putRequest(
+            createAction(UPDATE_RATING_TYPE),
+            createAction(RATING_TYPE_UPDATED),
+            `${window.API_BASE_URL}/api/v1/summits/${currentSummit.id}/selection-plans/${currentSelectionPlan.id}/track-chair-rating-types/${entity.id}?access_token=${accessToken}`,
+            normalizedEntity,
+            authErrorHandler,
+            entity
+        )({})(dispatch)
+            .then((payload) => {
+                dispatch(showSuccessMessage(T.translate("edit_rating_type.rating_type_saved")));
+            });
+
+    } else {
+        const success_message = {
+            title: T.translate("general.done"),
+            html: T.translate("edit_rating_type.rating_type_created"),
+            type: 'success'
+        };
+
+        postRequest(
+            createAction(UPDATE_RATING_TYPE),
```

Comment:
why u trigger this action, just set null

Reply:
done